### PR TITLE
Magus serial MIDI

### PR DESCRIPTION
**Pull request by antisvin**
This enables serial MIDI on Magus. RX is not usable with EXT port, but OWL FW doesn't have a mode for TX-only MIDI, so it would be trying to initialize it either way. I'm not sure if we actually want to get rid of RX or just leave that as an option for making hardware mods.

### CODE REVIEW

File: MagusEncoder/Makefile [hunk 1]
```diff
@@ -9,3 +9,6 @@ C_SRC = $(wildcard Src/*.c)
 CPP_SRC = $(wildcard Src/*.cpp)
 
 include $(OPENWARE)/Hardware/f0.mk
+
+upload:
+	$(OPENOCD) -c "program Build/${PROJECT}.elf verify reset exit"
```
Comment (pingdynasty) [inline MagusEncoder/Makefile:14]: wrong PR? and is this different from `flash` target?

**Review by pingdynasty — APPROVED** [review]
looks great!

File: MagusEncoder/Makefile (hunk 1 above)
Comment (antisvin) [reply MagusEncoder/Makefile:14]: A bonus addition. And that's how I've realized that flashing was in common.mk . Ok, I'll get rid of this, but it wasn't obvious at all - it wasn't present when I've initially used tried building it, before project got upgraded.

Comment (antisvin) [reply MagusEncoder/Makefile]: Dropped that commit

**Pull request merged**